tests/unit_tests/assets/ssl_certs/README.md

```diff
@@ -0,0 +1,31 @@
+This directory contains SSL certs/keys that our tests use when exercising SSL functionality.
+
+Here's how they were generated:
+
+```bash
+openssl req -x509 \
+    -newkey rsa:2048 \
+    -keyout localhost.key \
+    -out localhost.crt \
+    -days 9999 \
+    -nodes \
+    -subj "/C=US/ST=CA/L=San Francisco/O=Weights & Biases/OU=IT/CN=localhost" \
+    -addext 'subjectAltName=IP:127.0.0.1'
+
+ln -s localhost.crt $(openssl x509 -hash -noout -in localhost.crt).0
+```
+
+I don't really understand all that, but here are the most valuable things I learned in the process of coming up with ^that process:
+
+- "Why do we need the `ln -s ...certificate hash???...`?
+
+    - Both `requests` and `httpx` allow you to use an env var to point to a directory full of certificates.
+        - Docs: [`REQUESTS_CA_BUNDLE`](https://requests.readthedocs.io/en/latest/user/advanced/#ssl-cert-verification), [`SSL_CERT_DIR`](https://www.python-httpx.org/environment_variables/#ssl_cert_dir).
```

Comment:
I think most of the time users set REQUESTS_CA_BUNDLE to a specific file as opposed to an entire directory.  I know there's also the SSL_CERT_FILE option.  Probably not a huge deal but in an ideal world we would support either.

Reply:
`test_uses_userspecified_custom_ssl_certs` asserts that both `REQUESTS_CA_BUNDLE=/path/to/my.cert` and `REQUESTS_CA_BUNDLE=/path/to/my-dir/` work!

I intend to add support for `SSL_CERT_FILE` and `SSL_CERT_DIR` in [my httpx PR](https://github.com/wandb/wandb/pull/4552) (tested [here](https://github.com/wandb/wandb/pull/4552/files#diff-d91d7eef1134b4a1cb33ba953d5d8135ba77436ba91b0b119359fdb4d4e4b355R121)); but if you'd like, I could fold those changes into this PR instead.

Reply:
Sweet, separate PR's is fine 👍